doc/man3/SSL_write.pod

```diff
@@ -17,6 +18,14 @@ SSL_write_ex() and SSL_write() write B<num> bytes from the buffer B<buf> into
 the specified B<ssl> connection. On success SSL_write_ex() will store the number
 of bytes written in B<*written>.
 
+SSL_sendfile() writes B<size> bytes from offset B<offset> in the file
+descriptor B<fd> to the specified SSL connection B<s>. This function provides
+efficient zero-copy semantics. SSL_sendfile() is available only when ktls is
```

Comment:
It looks like the `ktls` should be uppercased to `KTLS`

Reply:
I think it is best to use Kernel TLS here to avoid the abbreviation in the documentation.

Reply:
I agree.